docs/LDAP.rst

```diff
@@ -193,11 +193,22 @@ Here is the list of all available configuration properties for directory integra
    |                                                | childMembershipAttributeName are fully qualified        |                           |                                    |                                         |
    |                                                | LDAP DNs.                                               |                           |                                    |                                         |
    +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+-----------------------------------------+
-   | exo.ldap.groups.rootGroup                      | Root group to bind LDAP/AD groups, all LDAP/AD groups   | Any group id, ending      | ``/platform/*``                    |                                         |
+   | exo.ldap.groups.rootGroup                      | Root group to bind LDAP/AD groups, all LDAP/AD groups   | Any group id, ending      | ``/ldap-groups/*``                 |                                         |
    |                                                | will be available under this group. It must end         | with ``/``                |                                    |                                         |
    |                                                | with ``/``. Root group ("/") cannot be used.            |                           |                                    |                                         |
    +------------------------------------------------+---------------------------------------------------------+---------------------------+------------------------------------+-----------------------------------------+
-
+   | exo.group.platform.identity.type               | it defines the identity type of groups under /platform  | ``String``                | ``GROUP``                          |                                         |
```

Comment:
The table seems to be malformed, see https://github.com/exoplatform/exo-documentation/blob/23ac8da70cc5d1f908d045f8105ce1e15777aaa0/docs/LDAP.rst
VS
https://github.com/exoplatform/exo-documentation/blob/master/docs/LDAP.rst

to build it locally you can use the commands detailed here :
https://github.com/exoplatform/exo-documentation/pull/293

Reply:
The table header caused the warnings. I didn't change it in my commit. I fixed all table issues in the documentation to decrease build warnings
Some more work is needed to remove all warnings, maybe @hbenali could help block documentation build when such warnings are still present.

Reply:
sorry, it seems the same here https://github.com/exoplatform/exo-documentation/blob/181ecbb3eb566846c655d2a22d7a0eb05cb6093f/docs/LDAP.rst#configuration-reference

Reply:
No all tables are fixed, unless you are taliking about another issue. I built locally and I have no issue for the LDAP.rst file

Reply:
@boubaker IMO Ali is trying to fix the display issue of `6.0`, not the `master` branch. I will handle this by backporting Ali's table related fixes. Maybe I'm wrong :-)